app/assets/javascripts/lookup_keys.js

```diff
@@ -157,7 +157,7 @@ function changeCheckboxEnabledStatus(checkbox, shouldEnable) {
     $(checkbox).attr('disabled', null);
```

Comment:
this should either use `.removeAttr('disabled')` or `.prop('disabled', false)`, not sure which is better in this case